superset-frontend/packages/superset-ui-core/src/models/ExtensibleFunction.ts

```diff
@@ -22,9 +22,8 @@
  */
 
 export default class ExtensibleFunction extends Function {
+  // @ts-ignore
   constructor(fn: Function) {
-    super();
```

Comment:
Calling `new Function()` breaks the CSP rule that disallows using `eval` in scripts.
Extending `Function` is a nice hack that let's us make a class instance (in most cases it's a formatter) callable - for example instead of calling `formatter.format(value)` we call `formatter(value)`. Removing `super()` does not break that behaviour while also letting us avoid calling Function constructor.